Gemfile

```diff
@@ -29,4 +29,5 @@ end
 
 group :test do
   gem "capybara"
+  gem "pry-byebug"
```

Comment:
I am very picky with dependencies and keep to the minimum.

Besides, the feature here does not depend on this gem.

Reply:
Fair enough, I'll remove the commit 😉